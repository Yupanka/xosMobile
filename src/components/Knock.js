import React from 'react';
import { Text, View } from 'react-native';
import { styles } from './styles';

class Knock extends React.Component {
  render () {
    return (
      <View style={styles.container}>
        <Text style={styles.welcome}>Knock knock</Text>
      </View>
    );
  }
}

export default Knock;
