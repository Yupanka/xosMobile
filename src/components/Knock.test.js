import React from 'react';
import { shallow } from 'enzyme';
import Knock from './Knock';

describe('Knock', () => {
  it('renders as expected', () => {
    console.log('shallow');
    const wrapper = shallow(
      <Knock />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
