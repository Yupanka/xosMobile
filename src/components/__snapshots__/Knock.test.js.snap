// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Knock renders as expected 1`] = `
<ForwardRef(View)
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ForwardRef(Text)
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    Knock knock
  </ForwardRef(Text)>
</ForwardRef(View)>
`;
